dist/variables/ds4/icon-variables.less

```diff
@@ -239,6 +239,14 @@
 @icon-carousel-prev-height: @icon-chevron-left-height;
 @icon-carousel-next-width: @icon-chevron-right-width;
 @icon-carousel-next-height: @icon-chevron-right-height;
+@icon-arrow-right-bold-width: 22px;
+@icon-arrow-right-bold-height: 21px;
+
+// aliases
+@icon-cta-width: @icon-arrow-right-bold-width;
```

Comment:
Do we need these aliases? If we are using them in SVG icons, should we maybe add the in the styles of the modules?
I don't thin we want those icons to be standalone, they should only be used in those modules. So maybe we can have in cta-button something like:
```
.cta-button > svg {
   width: @icon-arrow-right-bold-width;
}
```

Reply:
Good call. I'm going to be going through all of the icon variables and mixins at the end and have a good clean up! If it's not being used anywhere else then I will move it like you say.